app/src/main/java/net/squanchy/service/firestore/model/schedule/FirestoreSchedule.kt

```diff
@@ -0,0 +1,51 @@
+package net.squanchy.service.firestore.model.schedule
+
+import java.util.Date
+
+class FirestoreSchedulePage {
+    lateinit var day: FirestoreDay
```

Comment:
why lateinit? Also why an empty constructor and just setters? Is it because of the way Firestore works?

Reply:
Yeah Firestore won't use the ctor really. It sets fields directly, which would then need to be all nullable (even though they may not be optional values). This seems a bit more terse and allows us to retain null-safety. It's safe as by the time we get hold of an instance, these are instantiated within the `toObject()` call that creates the instance.

Reply:
Guessed so, 👍